EddystoneBeacon.cpp

```diff
@@ -117,3 +144,35 @@ unsigned char EddystoneBeacon::compressURI(const char* uri, char *compressedUri,
 void EddystoneBeacon::loop() {
   this->poll();
 }
+/* loop() for TLM below:**/
+float temperature;
+int B=3975;                  //B value of the thermistor
+float resistance;
+unsigned long pdu_count = 0;
+void EddystoneBeacon::tempLoop(int analogPin) {
+  this->poll();
+  pdu_count++;
+  /** temperature **/
+  pinMode(analogPin,INPUT);
+  int a = analogRead(analogPin);
+  resistance=(float)(1023-a)*10000/a; //get the resistance of the sensor;
+  temperature=1/(log(resistance/10000)/B+1/298.15)-273.15;//convert to temperature via datasheet&nbsp;;
+  buffer_1[4] = ((float2fix(temperature))&0x0000FF00)>>8; //LSB
+  buffer_1[5] = (float2fix(temperature))&0x000000FF;  //MSB
+  //temperature
+  
+  unsigned long val = millis()/100;
+  ulong2adv(buffer,10, val);  //UP-TIME SINCE POWER ON OR REBOOT
+  ulong2adv(buffer,6, pdu_count);  //PDU COUNT
```

Comment:
My understanding is PDU count is number of advertising frames. If it's based on time setTLMData is called, I think this will be incorrect. Default advertising interval 100ms. So 10 frames will be advertised every 1 second.